pennylane_forest/qpu.py

```diff
@@ -106,6 +102,17 @@ def __init__(
         """Union[None, pyquil.ExecutableDesignator]: the latest compiled program. If parametric
         compilation is turned on, this will be a parametric program."""
 
+
+        if kwargs.get("parametric_compilation", False):
+            # Raise a warning if parametric compilation was explicitly turned on by the user
+            # about turning the operator estimation off
+
+            # TODO: Remove the warning and toggling once a migration to the new operator estimation
+            # API has been executed. This new API provides compatibility between parametric
+            # compilation and operator estimation.
+            warnings.warn("Parametric compilation is currently not supported with operator"
+                          "estimation. Operator estimation is being turned off.")
```

Comment:
How come this is showing up as a change, has the location been moved?

Reply:
I changed this to have the same format as on the master branch.